Source/cmMakefileExecutableTargetGenerator.cxx

```diff
@@ -632,8 +635,9 @@ void cmMakefileExecutableTargetGenerator::WriteExecutableRule(bool relink)
     // Use a link script.
     const char* name = (relink ? "relink.txt" : "link.txt");
     this->CreateLinkScript(name, real_link_commands, commands1, depends);
-    if(this->Makefile->IsOn("CMAKE_EXPORT_LINK_COMMANDS")){
-      this->CreateLinkScriptJSON(name, real_link_commands, dependsForJson);
+    if (this->Makefile->IsOn("CMAKE_EXPORT_LINK_COMMANDS")) {
```

Comment:
There are two places with this if. Please, consider both. This is second

Reply:
This is WriteExecutableRule and other...